eta/core/data.py

```diff
@@ -290,10 +290,14 @@ def attributes(self):
         '''Returns the list of attributes of the data record that are to be
         serialized, i.e., all attributes that are not in `excluded()`
 
+        Note that Serializable.attributes() is called to remove private
+        attributes (those starting with "_").
+
         Returns:
             the list of attributes to be serialized
         '''
-        return [a for a in vars(self) if a not in self.excluded()]
+        attr = super(BaseDataRecord, self).attributes()
```

Comment:
why is this necessary now? Did BaseDataRecord previously remove private attributes?